src/screens/WelcomeScreen.tsx

```diff
@@ -0,0 +1,384 @@
+    shouldAnimateRainbows,
+  ]);
+
+  const buttonStyle = useAnimatedStyle(() => ({
+    transform: [{ scale: createWalletButtonAnimation.value }],
+    zIndex: 10,
+  }));
+
+  const contentStyle = useAnimatedStyle(() => ({
+    transform: [
+      {
+        scale: contentAnimation.value,
+      },
+    ],
+  }));
+
+  const textStyle = useAnimatedStyle(() => ({
+    backgroundColor: calculatedColor.value,
+  }));
+
+  const createWalletButtonAnimatedStyle = useAnimatedStyle(() => ({
+    backgroundColor: colors.dark,
+    borderColor: calculatedColor.value,
+    borderWidth: ios ? 0 : 3,
+    width: 230 + (ios ? 0 : 6),
+  }));
+
+  const createWalletButtonAnimatedShadowStyle = useAnimatedStyle(() => ({
+    backgroundColor: calculatedColor.value,
+    shadowColor: calculatedColor.value,
+  }));
+
+  const onCreateWallet = useCallback(async () => {
+    analytics.track('Tapped "Get a new wallet"');
+    const operation = dangerouslyGetState().index === 1 ? navigate : replace;
+    operation(Routes.SWIPE_LAYOUT, {
+      params: { emptyWallet: true },
+      screen: Routes.WALLET_SCREEN,
+    });
+  }, [dangerouslyGetState, navigate, replace]);
+
+  const showRestoreSheet = useCallback(() => {
+    analytics.track('Tapped "I already have one"');
+    navigate(Routes.RESTORE_SHEET, {
+      userData,
+    });
+  }, [navigate, userData]);
+
+  useAndroidBackHandler(() => {
+    return true;
+  });
+
+  return (
+    <Container testID="welcome-screen">
+      <RainbowsBackground shouldAnimate={shouldAnimateRainbows} />
+      <ContentWrapper style={contentStyle}>
+        {android && IS_TESTING === 'true' ? (
+          // @ts-expect-error JS component
+          <RainbowText colors={colors} />
+        ) : (
+          // @ts-expect-error JS component
+          <MaskedView maskElement={<RainbowText colors={colors} />}>
+            <RainbowTextMask style={textStyle} />
+          </MaskedView>
+        )}
+
+        <ButtonWrapper style={buttonStyle}>
+          <RainbowButton
+            emoji="castle"
+            height={54 + (ios ? 0 : 6)}
+            onPress={onCreateWallet}
+            shadowStyle={createWalletButtonAnimatedShadowStyle}
+            style={createWalletButtonAnimatedStyle}
+            testID="new-wallet-button"
+            text={lang.t('wallet.new.get_new_wallet')}
+            textColor={colors.white}
+          />
+        </ButtonWrapper>
+        <ButtonWrapper>
+          <RainbowButton
+            darkShadowStyle={sx.existingWalletShadow}
+            emoji="old_key"
+            height={56}
+            onPress={showRestoreSheet}
+            shadowStyle={sx.existingWalletShadow}
+            style={[
+              sx.existingWallet,
+              { backgroundColor: colors.blueGreyDarkLight },
+            ]}
+            testID="already-have-wallet-button"
+            text={lang.t('wallet.new.already_have_wallet')}
+            textColor={colors.alpha(colors.blueGreyDark, 0.8)}
+          />
```

Comment:
christian made some changes to this screen that are already merged to the .js version of this file as https://github.com/rainbow-me/rainbow/pull/3434/files#diff-5b1569bcf62ccdac475084998a3f2edf9d94ecfaca4d92cd3a48864629a61afcR568-R588 that i don't see here

Reply:
Working on it